extractor/src/test/java/org/schabi/newpipe/extractor/services/DefaultSearchExtractorTest.java

```diff
@@ -20,6 +27,8 @@ public boolean isCorrectedSearch() {
         return false;
     }
 
+    public abstract List<MetaInfo> expectedMetaInfo() throws MalformedURLException;
```

Comment:
Make this return an empty list by default, to prevent having to point it out manually every time, like with `DefaultStreamExtractorTest`